### PR TITLE
terminalizer: add page

### PR DESCRIPTION
**Pull request by Aracki**
[Terminalizer](https://github.com/faressoft/terminalizer) is a utility tool for recording your terminal and making a gif out of it.

### CODE REVIEW

File: pages/common/terminalizer.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# terminalizer
+
+> Utility program which records your terminal and generate animated gifs or share a web player.
```
Comment (agnivade) [inline pages/common/terminalizer.md]: Let us avoid using "your". Also, what do you mean by "share a web player" ?

File: pages/common/terminalizer.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Create a global config directory:
+
+`terminalizer init`
+
+- Record your terminal and create a recording file:
```
Comment (agnivade) [inline pages/common/terminalizer.md]: Record _the_ terminal

File: pages/common/terminalizer.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`terminalizer record {{file_name}}`
+
+- Play a recorded file on your terminal:
```
Comment (agnivade) [inline pages/common/terminalizer.md]: on _the_ terminal

File: pages/common/terminalizer.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`terminalizer play {{file_name}}`
+
+- Render a recording file as an animated gif image:
+
+`terminalizer render {{file_name}}`
+
+- Upload a recording file and get a link for an online player:
```
Comment (agnivade) [inline pages/common/terminalizer.md]: Where does it upload ?

File: pages/common/terminalizer.md (hunk 1 above)
Comment (agnivade) [reply pages/common/terminalizer.md]: I think just calling them "videos" keeps things simple and clear.

File: pages/common/terminalizer.md (hunk 4 above)
Comment (agnivade) [reply pages/common/terminalizer.md]: Just mentioning the upload destination should be fine IMO.

File: pages/common/terminalizer.md (hunk 1 above)
Comment (Aracki) [reply pages/common/terminalizer.md]: LGTM

File: pages/common/terminalizer.md (hunk 4 above)
Comment (Aracki) [reply pages/common/terminalizer.md]: I simplified it...

File: pages/common/terminalizer.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# terminalizer
+
+> Utility program which records the terminal and generate animated gifs or share a video.
+
+- Create a global config directory:
```
Comment (sbrl) [inline pages/common/terminalizer.md]: Does this create said directory in a pre-defined location, or in the current directory?

Comment (Aracki) [reply pages/common/terminalizer.md]: Default location is `$HOME/.terminalizer`. I think it's known that all "global" configs goes into `$HOME`.

Comment (Aracki) [reply pages/common/terminalizer.md]: Or you should think that need to be mentioned?

Comment (sbrl) [reply pages/common/terminalizer.md]: Ah, I see. I don't think we need specify the directory it puts it in. What about tweaking it to this?

```suggestion
- Create the global config directory:
```

Using the definite article here helps imply that we're talking about 1 specific instance, instead 1 out of potentially many instances.

**Pull request merged**